content/en/blog/2024/prom-and-otel/index.md

```diff
@@ -0,0 +1,564 @@
+under the [Cloud Native Computing Foundation (CNCF)](https://www.cncf.io/) 
+umbrella – but what role does each play in observability? 
+
+OpenTelemetry (OTel for short), is a vendor-neutral open standard for 
+instrumenting, generating, collecting, and exporting telemetry data. Prometheus 
+is a fixture of the observability landscape, widely relied upon for monitoring 
+and alerting within organizations. 
+
+While both Prometheus and OTel emit metrics, there is a lot to cover on the 
+differences and similarities, and is out of scope for this article. Rather, we 
+want to show you how OTel supports Prometheus, specifically in a Kubernetes 
+environment. You will learn:
+* How the OTel Collector's Prometheus Receiver can be used to ingest Prometheus 
+metrics.
+* How the OTel Operator's Target Allocator (TA) can be used for Prometheus 
+service discovery. 
+* How the TA ensures even distribution of Prometheus targets. 
+* Alternative methods for Prometheus metric collection through OTel-native 
+options such as the K8s cluster receiver and Kubelet stats receiver.  
+
+## Prometheus
+Prometheus encompasses many things, including [a time-series data store, 
+monitoring and alerting tool](https://prometheus.io/), and [data format](https://grafana.com/docs/grafana/latest/fundamentals/intro-to-prometheus/), known as [Prometheus 
+text-based exposition format](https://prometheus.io/docs/instrumenting/exposition_formats/#exposition-formats).
+
+Prometheus _data_ is stored as a dimensional time-series, meaning that the data 
+has attributes (i.e. labels/dimensions) and a timestamp. 
+
+The Prometheus _tool_ collects Prometheus metrics data from targets defined in a 
+configuration file. A target is an endpoint that supplies metrics for the 
+Prometheus tool to store.
+
+Prometheus is so ubiquitous in the monitoring space that many tools natively 
+emit metrics in Prometheus format, including [Kubernetes](https://kubernetes.io/docs/concepts/cluster-administration/system-metrics/) and [HashiCorp's Nomad](https://developer.hashicorp.com/nomad/docs/operations/monitoring-nomad). And for those that don’t, there are a number 
+of vendor- and community-built Prometheus exporters to aggregate and import data 
+into Prometheus.
+
+While you can use Prometheus to monitor a variety of infrastructure and 
+application metrics, one of its most popular use cases is to monitor Kubernetes. 
+This is the aspect of Prometheus monitoring that we will focus on in this 
+article.
+
+## Prometheus Metrics with OpenTelemetry
+In this section, you will learn about a couple of OTel collector components that 
+demonstrate the interoperability between OTel and Prometheus.  
+
+First, let’s do a quick refresher on the [collector](https://opentelemetry.io/docs/collector/) 
+– it’s an OTel component that can be used to collect and export data to and from 
+multiple sources and destinations. It also handles additional telemetry 
```

Comment:
"To and from multiple sources and destinations" sounds like the wrong way around:
```suggestion
– it’s an OTel component that can be used to collect and export data from and to 
multiple sources and destinations, respectively. It also handles additional telemetry 
```